erpnext/accounts/doctype/purchase_invoice/purchase_invoice.js

```diff
@@ -25,6 +25,12 @@ erpnext.accounts.PurchaseInvoice = erpnext.buying.BuyingController.extend({
 				this.frm.set_df_property("credit_to", "print_hide", 0);
 			}
 		}
+
+		// Trigger supplier event on load if supplier is available
+		// The reason for this is PI can be created from PR or PO and supplier is pre populated
+		if (this.frm.doc.supplier) {
+			this.frm.trigger('supplier');
```

Comment:
This is not working when you make purchase invoice using get items from purchase order

<img width="1199" alt="Screenshot 2020-09-08 at 10 08 54 AM" src="https://user-images.githubusercontent.com/8780500/92433686-655d1980-f1bb-11ea-9f1b-4d5e1dfab188.png">